src/librustc_trans/trans/context.rs

```diff
@@ -856,6 +855,29 @@ impl<'b, 'tcx> CrateContext<'b, 'tcx> {
             codegen_items.insert(cgi, TransItemState::NotPredictedButGenerated);
         }
     }
+
+    /// Given the node-id of some local item that has no type
+    /// parameters, make a suitable "empty substs" for it.
+    pub fn empty_substs_for_node_id(&self, item_node_id: ast::NodeId)
```

Comment:
I don't think we want this function, all callers *already* compute `local_def_id` themselves.